### PR TITLE
Fix NumBlockers

### LINKED ISSUE #1776: Spined Sliver ability does not calculate number of blockers on resolution
I blocked Spined Sliver and sacced the blocking creature while its trigger was on the stack. However, it resolved and still gave it +1/+1. This is wrong as even though it was blocked, there were 0 blocking creatures at resolution.

I came up against a quest deck with a bunch of similar abilities, so it may not be only for Spined Sliver, but I only observed this case.

### PR DESCRIPTION
**Pull request by tool4ever**
Closes #1776

### CODE REVIEW

File: forge-gui/res/cardsfolder/e/elvish_reclaimer.txt [hunk 1]
```diff
@@ -2,8 +2,7 @@ Name:Elvish Reclaimer
 ManaCost:G
 Types:Creature Elf Warrior
 PT:1/2
-S:Mode$ Continuous | Affected$ Card.Self | AddPower$ 2 | AddToughness$ 2 | CheckSVar$ X | SVarCompare$ GE3 | Description$ CARDNAME gets +2/+2 as long as there are three or more land cards in your graveyard.
-SVar:X:Count$TypeInYourYard.Land
+S:Mode$ Continuous | Affected$ Card.Self | AddPower$ 2 | AddToughness$ 2 | IsPresent$ Land.YouOwn | PresentZone$ Graveyard | Description$ CARDNAME gets +2/+2 as long as there are three or more land cards in your graveyard.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/e/elvish_reclaimer.txt:5]: I think this needs a PresentCompare

Comment (tool4ever) [reply forge-gui/res/cardsfolder/e/elvish_reclaimer.txt:5]: good spotting, thanks

**Comment by tool4ever** [comment]
I fixed a lot of LKI on triggers

**Review by Agetian — APPROVED** [review]

**Comment by Agetian** [comment]
A very nice fix/cleanup imo! :+1:

**Pull request merged**